app/controllers/candidate_interface/decisions_controller.rb

```diff
@@ -26,7 +26,9 @@ def respond_to_offer
       end
     end
 
-    def decline_offer; end
+    def decline_offer
+      @respond_to_offer = false
```

Comment:
Why was this added?

Reply:
Rails doesn't allow us to pass nil to the `model` argument anymore when using form_for anymore. It allows us to pass false though.

But we can also just remove the model attribute which I did here 09a6d1b7bd33fb145527e4816e8a244072ab3c47

Reply:
I think I can drop the commit that adds the false values though

Reply:
I think you might have. I went commit by commit so I probably missed it.